src/main/java/com/vmware/vipclient/i18n/base/instances/TranslationMessage.java

```diff
@@ -130,7 +130,12 @@ private String getMessageWithArgs(String resourceBundle, final Locale locale, fi
                 return FormatUtils.formatMsg(source, LocaleUtility.getSourceLocale(), args);
             }
         }
-
+        if(locale == null){
```

Comment:
Don't need `"".equalsIgnoreCase(locale)`?

Reply:
Yes, because it's a Locale type object, not a String type object.